Lectures/Lecture7_Reproducibility.md

```diff
@@ -1,35 +1,104 @@
++ Last updated: 29 08 2024
 + Lecturer: e.g. @SanliFaez 
 
 #### Contributors: 
-+ Erik van Sebille @erikvansebille
-+ Hosein Nemati
++ Sjoerd Terpstra @TerpstraS
 
 ## Development goal
-__why are we covering this topic and how can this lecture help participants progress towards that learning goal __
+Obtaining an intrinsic motivatio to documenting the project for replication and reproducibility
+
+![Project documentation illustrated as a roadmap](reproducibility.jpg)
+[The Turing Way project](https://the-turing-way.netlify.app/reproducible-research/reproducible-research) illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: 10.5281/zenodo.3332807.
+
+Student teams start with their measurement projects. They will also define the goal of their experiment and the analysis procedures, and create the proper documentation folder.
 
 ## Motivation: 
-__sketch the context of the lecture, why is this topic timely or urgent to cover, and what is desired or unwanted in the status quo.__
+Journaling and reporting, to self or colleagues, are a essential part of the scientific inquiry and an undismissible part of the credibility cycles. 
+Translating the higher goals of open science to daily practices of each researcher means documentation should be taken very seriously and thoroughly.
+
+Despite practicing reporting from high school in almost every project, several aspects of writing a report aimed at reproducibility stay neglected. Here are some
+anecdotes from the feedback that student gave each other last year
+
++ 'This should be like following a recipe to bake a cake, not like a investigation into a crime.' It took far too much effort to figure out what you did and then how you did what you did for anyone to WANT to reproduce it.
++ No issues whatsoever in reproducing.
++ We got stuck, because useful information that was present in the project description of the Git was not present in the README project.
+
++ Everything is in the expected files. The project documentation structure is clear.
++ It is the first thing the eyes of the reader fall on. The provided information is concise, but precise as well
++ They encouraged us and the procedure is easy enough, although the measurements and the analysis itself was not really motivating.
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
+By following the review exercise, each student will look criticaly at the documentation written by others and from the feedback they recieve will feel how their own documentation could have been imrpoved.
+
+Good documentation is mostly an exercise in empathy, and practicing that is the main goal of the exercise of this lecture. 
 
 ## Embedding/community
 __please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
```

Comment:
Missing the embedding/community. The community consists of people wanting to make use of a product (e.g., research output, software, lecture material), or groups using git to collaborate on projects.